pkg/server/common/const.go

```diff
@@ -69,6 +69,12 @@ const (
 	// AnnotationStorageUsage is annotation to store storage usuage information
 	AnnotationStorageUsage = "storage.cyclone.io/usage"
 
+	// AnnotationOwner is the annotation key used to indicate the owner of resources.
+	AnnotationOwner = "cyclone.io/owner"
```

Comment:
I suggest to use `Creator` instead of `Owner`, otherwise it would confuse with owner of resources in k8s.

Reply:
If you support user system later, users can change the owner, but the creator is fixed, not suitable to change.